__tests__/app.test.js

```diff
@@ -96,3 +100,23 @@ describe.only("GET /api/users", () => {
       });
   });
 });
+
+describe.only('PATCH /api/articles/:article_id', () => {
+  test("status 200 responds with updated article", () => {
+    return request(app)
+    .patch('/api/articles/4')
```

Comment:
underneath `.patch` we should have a `.send()` where we would pass the object containing new votes 
`{ inc_votes : 100 }` 
this would then be accessible on `req.body` which we can extract and use in our controller/model